src/components/cylc/workflow/Toolbar.vue

```diff
@@ -117,28 +117,35 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
           </v-list>
         </v-menu>
       </v-btn>
-      <router-link
-          to="/user-profile"
-        >
-      <v-tooltip>
-        <template v-slot:activator="{ props }">
-          <v-avatar v-bind="props"
-          class="ma-8 pa-2"
-          color="blue">
-            {{userInitials}}
-          </v-avatar>
-        </template>
-        <div class="text-h5">{{this.user.username}}</div>
-        <div class="d-flex justify-space-between" ><span class="mr-4">Admin</span> <span>{{this.user.admin}}</span></div>
-      </v-tooltip>
-    </router-link>
+      <v-btn icon size="small">
+        <v-avatar color="primary" size="small">
+          {{ userInitials }}
+        </v-avatar>
```

Comment:
I've converted the tooltip into a menu, as it is a common pattern for clicking the profile pic to open a menu rather than redirect you away from the page you were on, which could be unexpected for the user